presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetColumnIOConverter.java

```diff
@@ -40,7 +40,7 @@
 import static org.apache.parquet.io.ColumnIOUtil.columnRepetitionLevel;
 import static org.apache.parquet.schema.Type.Repetition.OPTIONAL;
 
-final class ParquetColumnIOConverter
+public class ParquetColumnIOConverter
```

Comment:
keep it `final`

in intellij we use `Java | Class structure | Utility class is not 'final'` inspection.

(obviously, we should document the list of inspections to use and to be disabled)